src/ng/directive/ngTransclude.js

```diff
@@ -159,35 +159,45 @@
  * </example>
  */
 var ngTranscludeMinErr = minErr('ngTransclude');
-var ngTranscludeDirective = ngDirective({
-  restrict: 'EAC',
-  link: function($scope, $element, $attrs, controller, $transclude) {
+var ngTranscludeDirective = ['$compile', function($compile) {
+  return {
+    restrict: 'EAC',
+    terminal: true,
```

Comment:
Since linking is a sync process (at least as far as ngTransclude is concerned), I'm pretty sure you shouldn't be able to observe the uncompiled content.

The timing of the pre and post linking functions may have changed in relation to to cousin or uncle directives (yay DOM family!), however I believe that the timing should remain unchanged in relation to ancestor directives which is the more important part.